projects/experimental/components/thumbnail-card/thumbnail-card.component.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    HostBinding,
+    Inject,
+    Input,
+} from '@angular/core';
+import {
+    TUI_INPUT_CARD_OPTIONS,
+    TuiInputCardOptions,
+    TuiPaymentSystem,
+} from '@taiga-ui/addon-commerce';
+import {TuiSizeL, TuiSizeS} from '@taiga-ui/core';
+
+@Component({
+    selector: 'tui-thumbnail-card',
+    templateUrl: './thumbnail-card.template.html',
+    styleUrls: ['./thumbnail-card.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TuiThumbnailCardComponent {
+    @Input()
+    @HostBinding('attr.data-size')
+    size: TuiSizeL | TuiSizeS = 'm';
+
+    @Input()
+    paymentSystem: TuiPaymentSystem | null = null;
+
+    @Input()
+    iconLeft = '';
+
+    @Input()
+    iconRight = '';
```

Comment:
what if some people pass Polymorpheous content?

Reply:
No reason to do so. Custom cases can use `ng-content`